.gitlab/tests/test.yml

```diff
@@ -39,7 +39,7 @@ workflow:
         ARCHCONFIG: llnl-cluster
         SCHEDULER_PARAMETERS: -N 1 -t 00:20:00 --exclusive --reservation=ci
         BENCHMARK: [kripke]
-        VARIANT: ['+strong~single_node caliper=mpi,time']
+        VARIANT: ['+strong~single_node caliper=mpi,time hwloc=on']
```

Comment:
The reason behind adding it to this test is because hwloc depends on Caliper, so better to save time to use this one already instead of having a new test so to have less test execution time and resources.